### ISSUE TITLE
Unable to finish install.

### ISSUE THREAD

**Issue by SavageAUS** [seven-of-eleven/Lenovo-ThinkPad-L13-Yoga-Hackintosh#34]
I have used your EFI on my 20R6 L13 and I can boot the installed and format the drive, and go through the first stage but after the first reboot I get an almost instant reboot at EB LOG RESET:RECOVERY.
Bios is set as per your repo so i do not understand.
I am trying to install Sonoma.

**Comment by seven-of-eleven** [comment]
Hi @SavageAUS if you are doing a fresh install of Sonoma it is best to disable the WiFi and Bluetooth kexts during the install, and then re-enable them after the install is complete.  Also ensure the `Misc>Security>SecureBootModel` to `Disabled` for the install. You can update it as well once the install has completed.

Give that a try and let me know how you make out.

Best of luck.

**Comment by SavageAUS** [comment]
I have managed to get it all installed using the bare minimum. 
Now it's just a matter of tinkering until everything works. 
It also looks like the bios has a whitelist for wifi? I have my Broadcom chip in and it is not detected anywhere.

**Comment by xtreme1690** [comment]
Yeah the system has a whitelist for the wifi. If you put something like a broadcom card in there then the wifi just stays disabled unfortunately and in the firmware settings there wont even be an option for wifi at all. I am yet to find a firmware mod that gets rid of this like you can on some other systems :/

**Comment by Mifaxav** [comment]
@SavageAUS @xtreme1690 its not a whitelist, the slot is a CNVi only slot. Bluetooth works because its using the USB protocol. There is NO fix for Broadcom, it just wont work.

**Comment by Mifaxav** [comment]
Also, try the boot arg amfi=0x80 with no BT/WiFi kexts

**Comment by simonsalib** [comment]
I've the same issue,
Is the disabling of the WiFi/Bluetooth fixing the issue?
